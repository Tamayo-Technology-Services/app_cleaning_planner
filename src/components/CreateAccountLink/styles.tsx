import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
  text: {
    fontSize: 12,
    color: 'black',
    fontWeight: '300',
  },
  createAccountText: {
    top: 2,
    fontSize: 12,
    color: '#0070F4',
    fontWeight: '300',
  },
})