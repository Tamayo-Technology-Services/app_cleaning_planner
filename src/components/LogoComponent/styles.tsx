
import { StyleSheet } from 'react-native';

export const loginStyles = StyleSheet.create({
logoContainer: {
    alignItems: 'center',
    marginBottom: 30,
  },
  logoText: {
    color: 'black',
    fontSize: 48,
    fontWeight: 'bold',
  },
  logo: {
    resizeMode: 'contain',
  },
})