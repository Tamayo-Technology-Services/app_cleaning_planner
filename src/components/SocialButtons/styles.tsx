import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
  socialButtonContainer: {
    flexDirection: 'row',
    justifyContent: 'space-between',
    marginBottom: 10,
    marginLeft: 15,
    marginRight: 15,
  },
  socialButton: {
    alignItems: 'center',
    marginLeft: 25,
    marginRight: 10,
    backgroundColor: '#fff',
    paddingVertical: 10,
  
    borderColor: '#F1F1F1',
    borderWidth: 1,
    borderRadius: 8,
    flex: 1,
  },
  socialButtonText: {
    color: 'white',
    fontSize: 16,
    textAlign: 'center',
  },
  socialButtonImage: {
    width: 13,
    height: 13,
    resizeMode: 'contain',
  }
})