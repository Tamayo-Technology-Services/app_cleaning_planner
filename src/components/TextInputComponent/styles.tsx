
import { StyleSheet } from 'react-native';

export const styles = StyleSheet.create({
input: {
    color: 'black',
    width: '100%',
    height: 40,
    borderColor: 'gray',
    borderWidth: 1,
    borderRadius: 5,
    marginBottom: 10,
    paddingHorizontal: 10,
  },
})