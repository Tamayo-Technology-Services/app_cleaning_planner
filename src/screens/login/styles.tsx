import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
    container: {
    flex: 1,
    justifyContent: 'center',
    // alignItems: 'center',
    padding: 10,
  },
  input: {
    width: '80%',
    height: 40,
    marginVertical: 10,
    borderWidth: 1,
    padding: 10,
  },
  forgotPasswordText: {
    textAlign: 'right',
    right: 20,
    marginBottom: 20,
    fontSize: 12,
    fontWeight: '300',
  },
  containerButton: {
    justifyContent: 'center', 
    alignItems: 'center'
  },
  loginButton: {
    width: '35%',
    backgroundColor: '#0070F4',
    paddingVertical: 10,
    borderRadius: 7,
  },
  loginButtonText: {
    color: 'white',
    fontSize: 12,
    fontWeight: 'bold',
    textAlign: 'center',
  },
  orText: {
    color: 'black',
    left: 30,
    marginTop: 20,
    marginBottom: 10,
    fontSize: 12,
    fontWeight: '300'
  },
  loginText:{
    fontWeight: 'bold',
    textAlign: 'left',
    paddingLeft: 30,
    paddingBottom: 10,

  }
})
