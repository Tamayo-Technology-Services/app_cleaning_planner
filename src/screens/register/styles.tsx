import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
    container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    padding: 10,
  },
  input: {
    width: '80%',
    height: 40,
    marginVertical: 10,
    borderWidth: 1,
    padding: 10,
  },
  forgotPasswordText: {
    textAlign: 'right',
    marginBottom: 20,
    color: 'blue',
  },
  loginButton: {
    backgroundColor: 'blue',
    paddingVertical: 15,
    paddingHorizontal: 50,
    borderRadius: 25,
  },
  loginButtonText: {
    color: 'white',
    fontSize: 18,
    textAlign: 'center',
  },
  orText: {
    color: 'black',
    marginTop: 20,
    marginBottom: 10,
    fontSize: 18,
  },
})