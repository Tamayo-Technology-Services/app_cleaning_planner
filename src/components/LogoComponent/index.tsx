import React from 'react';
import { View, Text, Image } from 'react-native';
import { loginStyles } from './styles';

const LogoComponent: React.FC = () => {
  return (
    <View style={loginStyles.logoContainer}>
      {/* <View style={styles.logoContainer}>
      </View> */}
      <Image source={require('./../../assets/images/bw.png')} style={loginStyles.logo} />
    </View>
  );
};

export default LogoComponent;