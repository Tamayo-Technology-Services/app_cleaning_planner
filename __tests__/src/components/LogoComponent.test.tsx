import React from 'react';
import '@testing-library/jest-native/extend-expect';
import { render } from '@testing-library/react-native';
import LogoComponent from '../../../src/components/LogoComponent';
import { loginStyles } from '../../../src/components/LogoComponent/styles';

describe('LogoComponent', () => {
  test('renders correctly', () => {
    const { getByText } = render(<LogoComponent />);
    const logoText = getByText('Logo');
    expect(logoText).toBeTruthy();
  });

  test('matches snapshot', () => {
    const { toJSON } = render(<LogoComponent />);
    expect(toJSON()).toMatchSnapshot();
  });

  test('applies styles correctly', () => {
    const { getByText } = render(<LogoComponent />);
    const logoText = getByText('Logo');
    expect(logoText).toHaveStyle(loginStyles.logoText);
  });
});