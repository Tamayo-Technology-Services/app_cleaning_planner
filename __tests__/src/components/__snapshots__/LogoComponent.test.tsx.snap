// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoComponent matches snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "marginBottom": 30,
    }
  }
>
  <Text
    style={
      {
        "color": "black",
        "fontSize": 48,
        "fontWeight": "bold",
      }
    }
  >
    Logo
  </Text>
</View>
`;
